myapp/spec/models/user_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+RSpec.describe 'Userモデルのテスト', type: :model do
+  name_max_length = 50
+  let(:name) { 'testUser' }
+  let(:email) { 'test@gmail.com' }
+  let(:password_digest) { 'testpass' }
+  let(:admin_flg) { 0 }
+  let!(:task) { User.new(name: name, email: email, password_digest: password_digest, admin_flg: admin_flg) }
```

Comment:
```suggestion
  let!(:user) { User.new(name: name, email: email, password_digest: password_digest, admin_flg: admin_flg) }
```

`User`モデルのテストなので、:userのほうがわかりやすいかと！

Reply:
確かにです！
見逃していました、ご指摘ありがとうございます！